### PR TITLE
Filter out non-api entries in breaking change documents

### PR DESCRIPTION
**Pull request by twsouthwick**

### CODE REVIEW

File: docs/BreakingChanges/037 New enum values in WPF's PageRangeSelection.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Edge
 Available
 
 ### Change Description
-Two new members (CurrentPage and SelectedPage) have been added to the <a href="https://msdn.microsoft.com/en-us/library/system.windows.controls.pagerangeselection(v=vs.110).aspx">PageRangeSelection</a> enum.
+Two new members (CurrentPage and SelectedPage) have been added to the [PageRangeSelection](https://msdn.microsoft.com/en-us/library/system.windows.controls.pagerangeselection(v=vs.110).aspx) enum.
```
Comment (mjrousos) [inline docs/BreakingChanges/037 New enum values in WPF's PageRangeSelection.md]: Do the embedded () need escaped or does common mark know how to handle those?

Comment (twsouthwick) [inline docs/BreakingChanges/037 New enum values in WPF's PageRangeSelection.md]: It's parsed fine

**Comment by mjrousos** [comment]
BreakingChangeParser.cs is showing up as a binary file :(

**Comment by twsouthwick** [comment]
Yeah :( We have a number of encoding issues apparently. I'm looking into do a clean sweep to convert everything to UTF8. At some point, some files got converted to UTF32

**Comment by mjrousos** [comment]
Would it be possible to convert this one to UTF8 prior to making your change so that I can at least see the diff in a commit?

**Comment by twsouthwick** [comment]
Sure

**Comment by twsouthwick** [comment]
Ok - I fixed all the files to be correct encoding: https://github.com/Microsoft/dotnet-apiport/commit/0c7cf2b119aee955b8e0b0bcb9c166a244633690

**Comment by mjrousos** [comment]
Thanks, @twsouthwick ! LGTM

**Pull request merged**